db/migrate/20230403091245_create_buses.rb

```diff
@@ -0,0 +1,13 @@
+class CreateBuses < ActiveRecord::Migration[7.0]
+  def change
+    create_table :buses do |t|
+      t.string :bus_name, null: false
+      t.string :bus_type, null: false
+      t.string :bus_brand, null: false
+      t.integer :bus_capacity, default:0, null: false
```

Comment:
`bus_capacity` related to seat in bus? is there any possiblity we defined `bus_capacity` 10 but we assigned more than 10 seat in that bus?

Reply:
Yes `bus_capacity` is related to `seat` in bus and i have handled that we cannot assigned more `seat` than `bus_capacity`